control/statesp.py

```diff
@@ -1331,3 +1331,105 @@ def ssdata(sys):
     """
     ss = _convertToStateSpace(sys)
     return ss.A, ss.B, ss.C, ss.D
+
+
+def consistent_dimensions(A=None, B=None, C=None, D=None):
```

Comment:
Should this be an internal function (`_consistent_dimensions`)?  Also, a more description function name might be useful.  Perhaps `_check_statespace_dimensions`?